PYME/Analysis/Tracking/trackUtils.py

```diff
@@ -452,7 +458,16 @@ def calcTrackVelocity(x, y, ci, t):
     #reorder
     v[I] = (1.0*v.copy())
 
-    return v
+    #edges
+    edges[1:] += (1.0-mask) # here we are looking for positive steps (i.e. np.diff(ci) >= .5)
+    edges[:-1] += (1.0-mask)
```

Comment:
possibly 'edges = (1.0 - w) > 0.5' would be even more compact but I am a little concerned that the meaning may be too obscure...